jmix-flowui/flowui-kit/src/main/java/io/jmix/flowui/kit/component/codeeditor/JmixCodeEditor.java

```diff
@@ -65,79 +68,140 @@ protected void initComponent() {
         }
 
         setInvalid(false);
+        setPrintMarginColumn(PRINT_MARGIN_COLUMN_DEFAULT_VALUE);
+        setFontSize(FONT_SIZE_DEFAULT_VALUE);
+
         addValueChangeListener(e -> validate());
         addClientValidatedEventListener(e -> validate());
     }
 
+    /** @see JmixCodeEditor#setHighlightActiveLine(boolean)
+     *
+     * @return Visibility of highlighting the active line
+     */
     @Synchronize(PROPERTY_HIGHLIGHT_ACTIVE_LINE)
     public boolean isHighlightActiveLine() {
         return getElement().getProperty(PROPERTY_HIGHLIGHT_ACTIVE_LINE, true);
     }
 
+    /**
+     * Sets the highlight of the line the cursor is on
+     */
     public void setHighlightActiveLine(boolean highlightActiveLine) {
         getElement().setProperty(PROPERTY_HIGHLIGHT_ACTIVE_LINE, highlightActiveLine);
     }
 
+    /**
+     * @see JmixCodeEditor#setShowGutter(boolean)
+     * @return Gutter visibility
```

Comment:
Well, brevity is the soul of wit, but let's write something more B2+, e.g. `whether gutter is visible` or `{@code true} if gutter is visible, {@code false} otherwise`.